creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -581,4 +611,54 @@ const _isReconfigEnabled = (enabledReconfigModes: string[], mode: string) => {
   return enabledReconfigModes.includes(mode)
 }
 
+type CanReconfigParams = {
+  libs: any
+  oldPrimarySpId: number
+  oldSecondary1SpId: number
+  oldSecondary2SpId: number
+  userId: number
+  logger: any
+}
+const _canReconfig = async ({
```

Comment:
mind naming this more specifically like `hasCorrectReplicaSet()`? this function is perfect and does one thing well, lets name it as such :)

Reply:
hmm @dmanjunath wanted it to be `_canReconfig`. i'll defer to you guys

Reply:
your call :)

Reply:
I will leave as is for simplicity 😬